api/v1alpha1/envoypatchpolicy_types.go

```diff
@@ -28,7 +28,7 @@ type EnvoyPatchPolicy struct {
 	Spec EnvoyPatchPolicySpec `json:"spec"`
 
 	// Status defines the current status of EnvoyPatchPolicy.
-	Status EnvoyPatchPolicyStatus `json:"status"`
```

Comment:
so this should be an pointer?

Reply:
good point, followed upstream APIs
https://github.com/kubernetes-sigs/gateway-api/blob/12dae809795674dfbca7b7637892a2f4027f6e80/apis/v1beta1/httproute_types.go#L43
but asked this question in the upstream slack channel, will get back on this ....

Reply:
I guess it's not a pointer because status is not needed when creating a new resource but it's still a necessary part of the resource.

Reply:
this shoule not be a block, we can fix it later.